### DISCUSSION TITLE
API for controlling playback

### DISCUSSION THREAD

**Discussion by ByRJavaDev** [tidal-music/discussions#124] [Q&A]
Hi All,

I have been checking the online developer documentation for Tidal to build a plugin for Touch Portal to integrate Tidal. The developer documentation only shows me functionality in 4 categories; Catalogue, Users, Playlist and Search. Going through them I can not find anything about actual simple playback such as play, pause, stop, volume up and down, next song, previous song, load song and so on.

I am probably looking at the wrong API, but I could not find other APIs or documentation for those functions. I did check out the SDKs for Web and Android and it seems to have these controls available. The problem however, is that I cannot use those, as the plugin I am writing is in Java and runs on desktops. 

Since that functionality is (seems) present, I assume there is some documentation somewhere to be found? Anybody can help me out to the correct place to find these or the ability to request them somehow?

**Comment by liamthexpl0rer** [comment]
I'm developing a python discord bot and have the same issue. I wish that the api becomes the ability to play music.

**Comment by stefanmoro** [comment]
Hey, as of now, playback is only supported through the SDKs available (documented on developer.tidal.com).

**Reply by ByRJavaDev** [reply]
Are there any plans to allow playback control on desktops in the future? Any dates?
